test/clj/rems/test_form_validation.clj

```diff
@@ -356,6 +356,37 @@
                                         :field/optional true
                                         :field/visible  true
                                         :field/value    "invalid.email"}]))))
+  
+    (testing "error: field input invalid phone number"
+      (is (= [{:type :t.form.validation/invalid-phone-number :field-id "2"}
+              {:type :t.form.validation/invalid-phone-number :field-id "3"}
+              {:type :t.form.validation/invalid-phone-number :field-id "4"}
+              {:type :t.form.validation/invalid-phone-number :field-id "5"}]
+             (validate-fields-for-draft [{:field/id       "1"
+                                          :field/type     :phone-number
+                                          :field/optional false
+                                          :field/visible  true
+                                          :field/value    ""}
+                                         {:field/id       "2"
+                                          :field/type     :phone-number
+                                          :field/optional false
+                                          :field/visible  true
+                                          :field/value    "+058450000100"}
+                                         {:field/id       "3"
+                                          :field/type     :phone-number
+                                          :field/optional false
+                                          :field/visible  true
+                                          :field/value    "+050050000100000000000000"}
```

Comment:
Is this one too long? doesn't it fail also because of the zero prefix?

Reply:
true, fixed that in the tests